lib/app/custom_widgets/events_widgets/substitution_event_widget.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/app_const_variables.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
+
+class SubstitutionEventWidget extends StatelessWidget {
+  const SubstitutionEventWidget({
+    super.key,
+    required this.playerIn,
+    required this.playerOut,
+    required this.isHomeTeam,
+  });
+
+  final String playerIn;
+  final String playerOut;
+  final bool isHomeTeam;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        if (isHomeTeam == true) ...[
+          const Image(
+            image: AssetImage(('${AppConstVariables.assetsPath}substitution.png')),
```

Comment:
Same here, you can create a separate `AssetsPaths` class and store these there.

Reply:
right, done that